source/_integrations/clickatell.markdown

```diff
@@ -3,6 +3,7 @@ title: Clickatell
 description: Instructions on how to add Clickatell notifications to Home Assistant.
 ha_category:
   - Notifications
+ha_iot_class: Cloud Polling
```

Comment:
```suggestion
ha_iot_class: Cloud Push
```

This is not a Cloud Polling integration, it's a Cloud Push integration since it performs a POST Request when there is a notification

Reply:
I thought it was more about where the data is coming from/who initiated the connection, Cloud Push from cloud, Cloud Polling from HA.
Will change it and check others.